src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.25.0
+
+7 October 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### New Features
+
+- [Announcing Okteto CLI 3.0](https://www.okteto.com/blog/cli-three-release/) - upgrading to chart release 1.25 requires a CLI upgrade to 3.0
+- Announcing Cloud Credentials, a central location to manage your authentication with AWS and GCP  <!-- 8550 -->
+- Enabling Okteto Insights for all users of Okteto <!-- 8627 -->
+- Added configurable buildkit network mode  <!-- 8486 -->
+- Init dynamic config  <!-- https://github.com/okteto/app/pull/8475 -->
```

Comment:
This is something internal. Not sure we should document it. Maybe we should approach it as Okteto installing a new CRD in your cluster? cc @pchico83

Reply:
Agree